src/views/open-api-docs-view-v2/server.ts

```diff
@@ -51,16 +52,18 @@ export async function getStaticProps({
 
 	/**
 	 * Fetch, parse, and validate the OpenAPI schema for this version.
+	 * Also apply any schema transforms.
 	 */
-	const rawSchemaData = await parseAndValidateOpenApiSchema(openApiJsonString)
-
-	/**
-	 * Apply any schema transforms.
-	 */
-	let schemaData = rawSchemaData
-	for (const schemaTransformFunction of schemaTransforms ?? []) {
-		schemaData = schemaTransformFunction(schemaData)
-	}
+	const schemaData = await parseAndValidateOpenApiSchema(
```

Comment:
Transforms need to be applied in the hook function argument provided by `parseAndValidateOpenApiSchema`, otherwise we end up dereferencing the schema _before_ applying transforms... which makes transforms that affect referenced definitions (such as shortening the `protobufAny` description) have no effect.